README.md

```diff
@@ -19,12 +19,24 @@ is not present in the Fedora repository or `stage-digital-object-set` cannot be
 the program terminates with an error.
 
 For each digital object the last version of each (managed) datastream, a `fo.xml` and `cfg.json` file are downloaded.
-The `fo.xml` file includes the inline datastreams DC, EMD, AMD, PRSQL, DMD.
-- the datastream AUDIT is skipped completely
-- RELS-EXT is exported into to the "relations"-map in the file `cfg.json` ([Digital Object Configuration])
+The `fo.xml` file includes the inline datastreams `DC`, `EMD`, `AMD`, `PRSQL`, `DMD` as far as they are present. As
+for the other datastreams:
+
```

Comment:
Use the asterisks for bulleted lists. This seems to be the standard way in markdown.